tools/functional/conftest.py

```diff
@@ -1,40 +1,122 @@
 import tempfile
+import time
+import uuid
 from pathlib import Path
 from random import randint
+from subprocess import check_output
 
 import pytest
 
 import gitlab
 
 
 TEMP_DIR = tempfile.gettempdir()
+TEST_DIR = Path(__file__).resolve().parent
```

Comment:
```suggestion
TEST_DIR = Path.cwd()
```

Reply:
Thanks! I had this as I am always a bit wary of using `cwd()` since I can't be sure where it's executed from. So I did the "derive path relative from the current script" dance 😁 I know It's a bit uglier but 100% safe. Currently with `tox -e py_func_v4` executed from the root dir this would give me an error because cwd is higher up (please don't laugh at the windows paths 😄):
```
ERROR tools/functional/api/test_gitlab.py::test_api_v4 - Exception: Command docker-compose -f "C:\Projects\python-gitlab\fixtures\docker-compose.yml" -p "pyt
est20584" up --build -d returned 1: """.FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Projects\\python-gitlab\\fixtures\\docker-compose.yml'
```
I'll check if pytest has some cleaner way for this.

Reply:
I guess that's okay, but pathlib should have some equivalent